doc/content/guides/interop-repository/_index.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "Interoperability repository"
+description: ""
+weight: 40
+---
+
+## Interoperability Repository
+
+In order to enable interoperability in the stack, create an interoperability configuration repository (e.g. a directory) with `config.yaml` at the root.
```

Comment:
Put somewhere the configuration settings to reference this file. Not everyone uses the Docker Compose file or knows to look there for settings